content/docs/tutorials/get-started/data-pipelines.md

```diff
@@ -0,0 +1,349 @@
+end-to-end.
+
+> See also the `dvc pipeline` command.
+
+## Visualize
+
+Now that we have built our pipeline, we need a good way to visualize it to be
+able to wrap our heads around it. DVC allows us to do that without leaving the
+terminal, making the experience distraction-less.
+
+> We are using the `--ascii` option below to better illustrate this pipeline.
+> Please, refer to `dvc pipeline show` to explore other options this command
+> supports (e.g. `.dot` files that can be used then in other tools).
+
+### Stages
+
+```dvc
+$ dvc pipeline show --ascii train.dvc
+     +-------------------+
+     | data/data.xml.dvc |
+     +-------------------+
+               *
+               *
+               *
+        +-------------+
+        | prepare.dvc |
+        +-------------+
+               *
+               *
+               *
+       +---------------+
+       | featurize.dvc |
+       +---------------+
+               *
+               *
+               *
+         +-----------+
+         | train.dvc |
+         +-----------+
+```
+
+### Commands
+
+```dvc
+$ dvc pipeline show --ascii train.dvc --commands
+          +-------------------------------------+
+          | python src/prepare.py data/data.xml |
+          +-------------------------------------+
+                          *
+                          *
+                          *
+   +---------------------------------------------------------+
+   | python src/featurization.py data/prepared data/features |
+   +---------------------------------------------------------+
+                          *
+                          *
+                          *
+          +---------------------------------------------+
+          | python src/train.py data/features model.pkl |
+          +---------------------------------------------+
+```
+
+### Outputs
+
+```dvc
+$ dvc pipeline show --ascii train.dvc --outs
+          +---------------+
+          | data/data.xml |
+          +---------------+
+                  *
+                  *
+                  *
+          +---------------+
+          | data/prepared |
+          +---------------+
+                  *
+                  *
+                  *
+          +---------------+
+          | data/features |
+          +---------------+
+                  *
+                  *
+                  *
+            +-----------+
+            | model.pkl |
+            +-----------+
+```
+
+## Reproduce
+
+Imagine a colleague just cloned the Git repo that has been created so far in
+another computer. This can be simulated by cloning our example-get-started repo
+from Github, and checking out the `7-train` tag:
+
+```dvc
+$ cd ~
+$ git clone https://github.com/iterative/example-get-started
+$ cd example-get-started
+$ git checkout 7-train
+```
+
+It's now extremely easy for anyone to reproduce the result end-to-end:
+
+```dvc
+$ dvc pull data/data.xml  # Get the initial raw data.
+$ dvc repro train.dvc     # Regenerate everything else.
+```
```

Comment:
Having to pull here is probably a problem... I think this happened because the data/data.xml.dvc has become an import stage in this new example proyect.

Reply:
Will be addressed for https://github.com/iterative/dvc.org/pull/1074#pullrequestreview-393503351.